src/indicator.js

```diff
@@ -385,6 +369,12 @@ var CPUFreqIndicator = new Lang.Class({
         return true;
     },
 
+    _getMinCheck: function()
```

Comment:
Good catch, I think we should reconsider the UI, but I'll be doing this with @fin-ger myself.